app/views/user_mailer/account_activation.html.erb

```diff
@@ -0,0 +1,5 @@
+<p><%= t "hi" %> <%= @user.name %>,</p>
+
+<p><%= t "welcome_to" %> <%= t "click_to_activate" %> </p>
+
+<%= link_to t("activate"), edit_account_activation_url(id: @user.activation_token, email: @user.email) %>
```

Comment:
Sao lại dùng _url mà không dùng path em ơi

Reply:
dạ vì  dùng _url sẽ cho toàn bộ đường dẫn, là đường dẫn tuyệt đối. do gửi link trong email nên phải cung cấp đầy đủ url ạ.